### PR TITLE
chore(artifacts): make sure is_sequence is set when loading artifact collection

### PR DESCRIPTION
**Pull request by estellazx**
Description
-----------
https://wandb.atlassian.net/browse/WB-24566

Testing
-------
local testing

### CODE REVIEW

File: wandb/apis/public/artifacts.py [hunk 1]
```diff
@@ -303,6 +304,8 @@ def __init__(
         self._tags = [a["node"]["name"] for a in self._attrs["tags"]["edges"]]
         self._saved_tags = copy(self._tags)
         self.organization = organization
+        if is_sequence is not None:
```
Comment (ibindlish) [inline wandb/apis/public/artifacts.py:307]: I don't think this param should override the assignment that comes through `load()`. maybe we should move it above line 299? wdyt?

Comment (ibindlish) [reply wandb/apis/public/artifacts.py:307]: Or we could ignore this parameter if `attr is None`?

**Comment by estellazx** [comment]
> lgtm, thanks! should we add a system test for this? to make sure you're able to edit a collection right after fetching it via the registry api?

I was planning on adding system tests for all of search registry API after the other milestones are in since none of it is system tested right now.
